state/accountsDB.go

```diff
@@ -131,7 +132,25 @@ func NewAccountsDB(
 		},
 		processingMode: processingMode,
 		lastSnapshot:   &snapshotInfo{},
-	}, nil
+	}
+
+	val, err := trieStorageManager.Get([]byte(common.ActiveDBKey))
```

Comment:
can we find the `common.ActiveDBKey` in other, old, storers but not in the current one?

Reply:
Yes. If the snapshot for the current epoch was started, but then the node was killed, it means that the current db will not have that key.